src/NServiceBus.Transport.RabbitMQ/RabbitMQTransport.cs

```diff
@@ -91,6 +93,11 @@ public TimeSpan TimeToWaitBeforeTriggeringCircuitBreaker
             }
         }
 
+        /// <summary>
+        /// The callback to use when customizing the message before dispatching it to the broker.
+        /// </summary>
+        public Action<IOutgoingTransportOperation, IBasicProperties> OutgoingNativeMessageCustomization { get; set; }
```

Comment:
Would it be possible to go the extra mile to describe the potential danger of this operation including the flow of things like transport customization gets applied first and then this thing is called which means you can manipulate things that might lead to things not working elsewhere?

Reply:
👍